app/lib/efile/line_data.yml

```diff
@@ -592,6 +592,14 @@ MD502_LINE_22:
   label: '22 Earned income credit (EIC)'
 MD502_LINE_22B:
   label: '22B Claiming the Maryland Earned Income Credit with a qualifying child.'
+MD502_LINE_23:
+  label: 'Poverty level credit'
+MD502_LINE_24:
+  label: 'Other income tax credits for individuals from Part AA, line 14 of Form 502CR'
```

Comment:
[dust] I've seen elsewhere noting that the label here notes that this is stubbed, so that it's clear that this shouldn't return anything.